### ISSUE TITLE
Feature Enhancement - Process multiple images, not just "main image"

### ISSUE THREAD

**Issue by moorecreative** [HotcakesCommerce/ProcessImagesModule#2]
We  have a need for this image processor to prepare image sizes, etc for multiple images for a product beyond the "main image" for that product. We will post notes and updates here if we get into it as an enhancement.

**Comment by WillStrohl** [comment]
Can you please provide more details about what you mean, specifically?  There are already multiple images created.  So I want to be sure I understand what you're asking for.  

https://hotcakescommerce.zendesk.com/hc/en-us/articles/205436695-Product-Image-Sizes

**Comment by moorecreative** [comment]
From what we can see when testing (and past usage), this can be used to bulk import/process the primary image for a product.

 

However, if you want to also process and have a media gallery set of 2nd-10th images for that product, then the current state of this module does not handle those.

 

Is there something that we missed about different ways to use this module or to name the files so that they do process as additional product images?

 

Ryan Moore

 <mailto:ryan@moorecreative.com> ryan@moorecreative.com

704-896-6057 office

**Comment by WillStrohl** [comment]
That is correct. It's always been that way and so far has only been a prrof of concept extension for Hotcakes. Now that the pattern is in place, though, it should be very easy for any developer to add the code required for processing additional images and submit a pull request. This update took the better part of 10 hours, but the remaining update you're asking about should take less than half that time. 😁

Sent from my T-Mobile 4G LTE Device
Get Outlook for Android<https://aka.ms/ghei36>

________________________________
